### PR TITLE
[one-cmds] Add PyTorch frontend

### PR DESCRIPTION
**Pull request by binarman**
This PR adds pytorch frontend to one-cmds.

related issue: #7939

ONE-DCO-1.0-Signed-off-by: Alexander Efimov <a.efimov@samsung.com>

### CODE REVIEW

File: compiler/one-cmds/one-import-pytorch [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+    # verify arguments
+    _verify_arg(parser, args)
+
+    # convert
+    _convert(args)
+
+
+if __name__ == '__main__':
+    main()
+    # _utils._safemain(main, __file__)
```
Comment (jyoungyun) [inline compiler/one-cmds/one-import-pytorch:356]: Is this comment-out what you intended?

**Comment by jyoungyun** [comment]
I don't have much knowledge of `pytorch`. However except for the part below, the implementation looks the same other import files. 

https://github.com/Samsung/ONE/blob/dab1cd230f78f472d80030229419d56466d3bbb6/compiler/one-cmds/one-import-pytorch#L254-L301

This code looks good to me. Could you check my comment?

File: compiler/one-cmds/one-import-pytorch (hunk 1 above)
Comment (binarman) [reply compiler/one-cmds/one-import-pytorch:356]: oops, thank you!

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Comment by binarman** [comment]
@mhs4670go PTAL?

also related PRs: #8063 and #8060

File: compiler/one-cmds/one-import-pytorch [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+    for shape_str in shapes_str.split(":"):
+        if shape_str != "":
+            shapes += [list(map(int, shape_str.split(",")))]
+        else:
+            shapes += [[]]
+    return shapes
+
+
+def _parse_types(types_str):
+    # There are no convenient way to create torch from string ot numpy dtype, so using this workaround
```
Comment (mhs4670go) [inline compiler/one-cmds/one-import-pytorch:139]:
```suggestion
    # There are no convenient way to create torch from string of numpy dtype, so using this workaround
```
Maybe?? This change could be applied later.

File: compiler/one-cmds/one-import-pytorch [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+                    opset_version=onnx_opset_version)
+                onnx_saved = True
+                break
+            except:
+                f.write(('attempt failed\n').encode())
+
+        if not onnx_saved:
+            raise ValueError('Failed to save temporary onnx model')
+
+        # convert onnx to tf saved mode
```
Comment (mhs4670go) [inline compiler/one-cmds/one-import-pytorch:303]:
```suggestion
        # convert onnx to tf saved model
```

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**